test/core/transport/binder/end2end/fuzzers/client_fuzzer.cc

```diff
@@ -0,0 +1,330 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include <thread>
+#include <utility>
+
+#include "absl/memory/memory.h"
+
+#include "src/core/ext/transport/binder/transport/binder_transport.h"
+#include "src/core/ext/transport/binder/wire_format/binder.h"
+#include "src/core/ext/transport/binder/wire_format/wire_reader.h"
+#include "src/core/lib/iomgr/executor.h"
+#include "src/core/lib/surface/channel.h"
+
+bool squelch = true;
+bool leak_check = true;
```

Comment:
not used?

Reply:
These are extern variables used in `test/core/util/fuzzer_corpus_test.cc`

Reply:
Ah ok. Maybe worth commenting? (Or is it already very obvious in this specific context so you didn't comment it?)

Reply:
Done.